content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,48 @@
+
+## How does OLM enable cluster admins and Developers?
+
+The OLM is enabled by default in OpenShift Container Platform 4.X, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The OpenShift Container Platform web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
+
+For developers, a self-service experience allows provisioning and configuring instances of databases, monitoring, and big data services without having to be subject matter experts, because the Operator has that knowledge baked into it.
+
+
+## Build with the Operator SDK
+
+[Operator SDK][osdk] is also a component of the [Operator Framework][operator-framework]. Writing an operator today can be difficult because of challenges such as using low-level APIs, writing boilerplate, and a lack of modularity which leads to duplication.
+The [Operator SDK][osdk] is a framework that uses the [controller-runtime][controller-runtime] library to make writing operators easier by providing:
+- High-level APIs and abstractions to write the operational logic more intuitively
+- Tools for scaffolding and code generation to bootstrap a new project fast
+- Extensions to cover common operator use cases
+
+Note that [Operator SDK][osdk] provides helpers and features to help you [integrate your project with OLM][olm-integration].  For further information check https://sdk.operatorframework.io/ .
+
+
+## Package with the Operator Lifecycle Manager
+
+With OLM, administrators can control which Operators are available in what namespaces and who can interact with running Operators. The permissions of an Operator are accurately configured automatically to follow a least-privilege approach. OLM manages the overall lifecycle of Operators and their resources, by doing things like resolving dependencies on other Operators, triggering updates to both an Operator and the application it manages, or granting a team access to an Operator for their slice of the cluster.
+
+Simple, stateless applications can use the Lifecycle Management features of the Operator Framework—without writing any code—by using a generic Operator (for example, the Helm Operator). However, complex and stateful applications are where an Operator can be especially useful. The managed-service capabilities that are encoded into the Operator code can provide an advanced user experience, automating such features as updates, backups and scaling.
```

Comment:
```suggestion
Simple, stateless applications can use the Lifecycle Management features of the Operator Framework — without writing any code — by using a generic Operator (for example, the Helm Operator). However, complex and stateful applications are where an Operator can be especially useful. The managed-service capabilities that are encoded into the Operator code can provide an advanced user experience, automating such features as updates, backups, and scaling.
```

Reply:
Hmm, nevermind this looked strange on this first read, but this renders fine when looking at the markdown preview tab of these changes. Disregard.